### ISSUE TITLE
Positive synovial Tropheryma whipplei bacteria microscopy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11157]
**Preferred term label:**
Positive synovial Tropheryma whipplei bacteria microscopy

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Similar to https://hpo.jax.org/browse/term/HP:6000766 but a different fluid site

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Whipple disease
Testing mentioned in clinical use in https://omim.org/entry/621097

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001431
- label: Positive synovial Tropheryma whipplei bacteria microscopy
- parents: [Positive synonvial fluid infectious agent test (HP:6000581)](https://hpo.jax.org/app/browse/term/HP:6000581)
- pmid: 
- def: Demonstration of Tropheryma whipplei bacteria in the synovial (joint) fluid by microscopic analysis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11157
- synonyms:
